.github/actions/json-syntax-validator/action.yml

```
@@ -0,0 +1,43 @@
+name: JSON Syntax Validator
+description: Validate the syntax of a list of JSON files
+inputs:
+  paths:
+    description: Comma separated list of files
+    required: true
+runs:
+  using: composite
+  steps:
+    - run: |
+        # do not stop the script if an error during file validation occurs
+        set +e
+
+        # split the input paths by commas or new lines
+        files=($(echo ${{ inputs.paths }} | tr "," "\n"))
+
+        # count of the invalid files
+        invalidFiles=0
+
+        for file in "${files[@]}"
+        do
+          echo "Validating file:" $file
+          jq empty $file
+
+          # if the JSON content of the file is not valid increase the counter
+          if [ "$?" -ne 0 ]
```

It should be safe to just do:
```suggestion
          if [ $? -ne 0 ]
```